stages/org.osbuild.users

```diff
@@ -125,6 +126,9 @@ def main(tree, options):
         else:
             useradd(tree, name, uid, gid, groups, description, home, shell, password, expiredate)
 
+        if password_changed_date is not None:
+            subprocess.run(["chroot", tree, "chage", "--lastday", str(password_changed_date), name], check=True)
```

Comment:
The `chage` command has a `--root` option which should work the same as `chroot`ing.  Not a big deal since we already `chroot` for the other commands, but something to keep in mind if we pull it out to another stage and want to minimise explicit use of `chroot`.

Reply:
Noted. I'll leave this unresolved until it's determined whether this belongs in its own stage.

Reply:
Resolving this conversation with no change because it seems like we don't a full stage just for `chage`.